packages/abi-gen/src/cli-format/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'cli-format';
```

Comment:
I think you can just add this to the `global.d.ts` file instead of having a separate `index.d.ts` just for this dep. Also, we typically try to write the minimal type defs for packages given the functionality we use.

Reply:
https://github.com/0xProject/0x-monorepo/pull/1878/commits/d6e965b205604d27ffcf5991cb25bf3481ae8ed0